guice-intro/src/main/java/com/baeldung/examples/RunGuice.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.baeldung.examples;
+
+import com.baeldung.examples.guice.Communication;
+import com.baeldung.examples.guice.binding.AOPModule;
+import com.baeldung.examples.guice.modules.BasicModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import java.util.Scanner;
+
+/**

```

Comment:
Please remove this JavaDoc from all files if @author is the only thing in it.
If any of them have other content in the JavaDoc, change the author to 'baeldung' or remove the author info.